config/samples/clusters_v1beta1_kafka.yaml

```diff
@@ -56,10 +56,10 @@ spec:
 #          diskEncryptionKey: "123e4567-e89b-12d3-a456-426614174000"
 #          resourceGroup: "asdfadfsdfas"
 #      privateLink:
-#        - advertisedHostname: "asdfadsf"
+#        - advertisedHostname: "bohdan-kafka-test.com"
```

Comment:
I don't think that it is appropriate to use names here

Reply:
changed to `kafka-example-test.com`